docs/migrating_to_swagger_20.rst

```diff
@@ -3,12 +3,13 @@ Migrating to Swagger 2.0
 
 So you're using pyramid_swagger with Swagger 1.2 and now it is time to upgrade to Swagger 2.0.
 
-Just set the version of Swagger to support via configuration like so:
+Just set the version of Swagger to support via configuration. Also, set ``use_models`` to ``false`` so that your existing Swagger 1.2 views will continue to received models in incoming requests as dicts instead of Python classes.
```

Comment:
The purpose and effect of this option are unclear to me. Is "use_models" going to modify the data bound to my request? Should this read "existing Pyramid views" not "Swagger 1.2"...if not what is the Swagger view being referred to?